clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1951,6 +1951,19 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
         };
     }
 
+    class CountingRebalanceListener implements ConsumerRebalanceListener {
```

Comment:
nit: maybe extract out the `MockRebalanceListener` out of ConsumerCoordinatorTest to be shared by multiple test classes?

Reply:
Ack good call